mmv1/third_party/terraform/data_sources/data_source_compute_region_network_endpoint_group.go.erb

```diff
@@ -0,0 +1,77 @@
+<% autogen_exception -%>
```

Comment:
These files have no ruby injected into them, so we should make them `.go` files instead of `.go.erb` files and remove this autogen exception line